### PR TITLE
docs(store/tables): update for MUD 2.0

### PR DESCRIPTION
**Pull request by qbzzt**
The biggest change is that this syntax doesn't compile for some reason:

```
FieldLayout fieldLayout = FieldLayoutLib.encode({
  staticFields: staticFields,
  numDynamicFields: 0
});
```

### CODE REVIEW

File: docs/pages/store/tables.mdx [hunk 1]
```diff
@@ -57,6 +61,9 @@ ResourceId offchainTableId = ResourceIdLib.encode({
   typeId: RESOURCE_OFFCHAIN_TABLE,
   name: "Balance"
 });
+
+console.log("Onchain table id:%x", uint256(ResourceId.unwrap(tableId)));
+console.log("Offchain table id:%x", uint256(ResourceId.unwrap(offchainTableId)));
```
Comment (holic) [inline docs/pages/store/tables.mdx:66]: is this leftover?

File: docs/pages/store/tables.mdx [hunk 2]
```diff
@@ -46,6 +46,10 @@ The table ID encodes the table's type and name in a 32-byte `ResourceId` value a
 import { ResourceId, ResourceIdLib } from "@latticexyz/store/src/ResourceId.sol";
 import { RESOURCE_TABLE, RESOURCE_OFFCHAIN_TABLE } from "@latticexyz/store/src/storeResourceTypes.sol";
 
+.
+.
+.
+
```
Comment (holic) [inline docs/pages/store/tables.mdx:52]: I don't think we need this?

File: docs/pages/store/tables.mdx (hunk 1 above)
Comment (qbzzt) [reply docs/pages/store/tables.mdx:66]: No, it's something I thought would be useful for people so I left it. I'll make that clearer.

File: docs/pages/store/tables.mdx (hunk 2 above)
Comment (qbzzt) [reply docs/pages/store/tables.mdx:50]: It signifies a change of context, from the header matter to inside a function inside the contract. I changed it to horizontal dots so it'll take less space.

Comment (qbzzt) [reply docs/pages/store/tables.mdx:50]: It signifies a change of context from the header stuff to inside a function inside the contract. I changed it to a horizontal ... so it'll take less space.

**Review by alvrs — APPROVED** [review]

**Pull request merged**